packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,301 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorInvalidCount, errorInvalidResponse, errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * @hidden
+ * Hide from docs
+ * --------
+ * Interact with image and video. It lets the app developer ask the user to get images or video from their camera / camera roll / file system.
```

Comment:
```suggestion
 * Interact with images. Allows the app developer ask the user to get images from their camera / camera roll / file system.
```

Removed references to video from the comment since video is not currently available.